tools/tkn-results/docs/tkn-results_records_get.md

```diff
@@ -0,0 +1,29 @@
+## tkn-results records get
+
+Get Record
+
+```
+tkn-results records get [flags] <record>
+
+  <record parent>: Record parent name to query. This is typically "<namespace>/results/<result name>", but may vary depending on the API Server. "-" may be used as <result name> to query all Results for a given parent.
+```
+
```

Comment:
Ditto - another auto-gen mismatch.

Reply:
No idea why this happened.  Ran the autogen command again.